backend/backend/settings.py

```diff
@@ -33,7 +33,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
We may change `DEBUG` to false. Whats your opinion?
```suggestion
DEBUG = False
```